### DISCUSSION TITLE
How to Import collection .db from Window Osu legacy to Mac Osu Lazer ?

### DISCUSSION THREAD

**Discussion by Valstrax04** [ppy/osu#24452] [Q&A]
Hello Guys, 

I have a collection with 100 songs on my Window with osu classic/legacy.

I have my mac now who run Osu Lazer perfectly and I would like to import all my collection.

I did export the .db collection file from my mac ( which weight 700mo ) 

But where should I place it ?

**Comment by alexpado** [comment]
As far as I know, there is no way to import only a subset of data. 
Your best bet would be:

1. Install osu!stable on your mac
2. Copy all songs/skins/collection.db from your windows installation to your mac installation
3. In lazer settings, "Run setup wizard" and follow instructions 

At this step, locate your osu!stable install directory and import your data.

![image](https://github.com/ppy/osu/assets/31733699/80260870-8e09-40ef-8176-59a5a6f457a7)

Maybe step 1 & 2 can be made into one by just copying the whole folder without installing osu!stable, but I'm not sure
